src/project/qml/MuseScore/Project/AlsoShareAudioComDialog.qml

```diff
@@ -0,0 +1,185 @@
+        }
+    }
+
+    onAccessibilityActivateRequested: {
+        accessibleInfo.readInfo()
+    }
+
+    NavigationPanel {
+        id: navPanel
+
+        name: "OptionsPanel"
+        order: 1
+        section: root.navigationSection
+        direction: MUAccessible.Horizontal
+        accessible.role: MUAccessible.Dialog
+
+        onNavigationEvent: function(event) {
+            if (event.type === NavigationEvent.AboutActive) {
+                var btn = buttonBox.firstFocusBtn
+                if (Boolean(btn) && btn.enabled) {
+                    event.setData("controlIndex", [ btn.navigation.row, btn.navigation.column ])
+                }
+            } else {
+                buttonBox.restoreAccessibility()
+                accessibleInfo.resetFocus()
+            }
+        }
+    }
+
+    AccessibleItem {
+        id: accessibleInfo
+
+        accessibleParent: navPanel.accessible
+        role: MUAccessible.Button
+        name: titleInfo.text + "; " + subtitleInfo.text + "; " + preferenceInfo.text
+
+        function readInfo() {
+            accessibleInfo.ignored = false
+            accessibleInfo.focused = true
+        }
+
+        function resetFocus() {
+            accessibleInfo.ignored = true
+            accessibleInfo.focused = false
+        }
+    }
+
+    ColumnLayout {
+        id: content
+
+        anchors.fill: parent
+        spacing: 20
+
+        ColumnLayout {
+            id: header
+
+            width: parent.width
+            spacing: 16
+
+            StyledTextLabel {
+                id: titleInfo
+
+                Layout.fillWidth: true
+
+                //: The text between `<a href=\"%1\">` and `</a>` will be a clickable link to Audio.com
+                text: qsTrc("project/cloud", "Would you also like to share your music on <a href=\"%1\">Audio.com</a>?").arg(root.audioComUrl)
+                font: ui.theme.largeBodyBoldFont
+            }
+
+            StyledTextLabel {
+                id: subtitleInfo
+
+                Layout.fillWidth: true
+
+                text: qsTrc("project/cloud", "Share your score's audio with millions of listeners on this free streaming platform")
+                font: ui.theme.bodyFont
+            }
+        }
+
+        Image {
+            Layout.fillWidth: true
+            Layout.preferredHeight: sourceSize.height * (width / sourceSize.width)
+            fillMode: Image.PreserveAspectFit
+
+            source: "qrc:/resources/AudioCom_Waveform.png"
+        }
+
+        StyledTextLabel {
+            id: preferenceInfo
+
+            Layout.fillWidth: true
+            horizontalAlignment: Text.AlignLeft
+
+            text: qsTrc("project/cloud", "You can change this setting in Preferences at any time.")
+            font: ui.theme.bodyFont
+        }
+
+        RowLayout {
+            id: options
+
+            CheckBox {
+                id: checkbox
+
+                Layout.fillWidth: true
+
+                navigation.panel: navPanel
```

Comment:
I think it should be in a separate panel than the ButtonBox and the user will be able to navigate via Tab.
Also please set `navigation.row`, at the moment this checkbox is not available for navigation as I see